definitions/ext-network_dhcp_service/definition.yml

```diff
@@ -0,0 +1,38 @@
+domain: EXT
+type: NETWORK_DHCP_SERVICE
+synthesis:
+  rules:
+  # IPv4 DHCP Service via Gigamon
+  - identifier: dst_ip
+    name: dst_ip
+    conditions:
+    - attribute: app_name
+      value: dhcp
```

Comment:
what is the reason to filter just by this app name?

Reply:
The Gigamon data source is a summary of the packet capture data and contains any number of target applications. Filtering to the app_name is Gigamon speak for _"Show just the DHCP-related packets"_ -- it's not New Relic speak for 'App'. Confusing, I know.